repos/spack_repo/builtin/packages/fairroot/package.py

```diff
@@ -0,0 +1,95 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+
+from spack.package import *
+
+
+class Fairroot(CMakePackage):
+    """C++ simulation, reconstruction and analysis framework for particle physics experiments"""
+
+    homepage = "http://fairroot.gsi.de"
+    url = "https://github.com/FairRootGroup/FairRoot/archive/v18.8.2.tar.gz"
+    git = "https://github.com/FairRootGroup/FairRoot.git"
+    maintainers("dennisklein", "fuhlig1", "jezwilkinson")
+
```

Comment:
```suggestion

    tags = ["hep"]
```
Adding a tag is a nice way to keep an overview of HEP packages in spack.